portal/server/api.js

```diff
@@ -797,22 +797,38 @@ const getUkefDecisionReport = async (token, payload) => {
   }
 };
 
-const uploadReportAndSendNotification = async (token, reportData) => {
+const uploadUtilisationReportData = async (uploadingUser, month, year, csvData, csvFileBuffer, token) => {
   try {
+    const formData = new FormData();
+    formData.append('reportData', JSON.stringify(csvData));
+
+    formData.append('user', JSON.stringify(uploadingUser));
+    formData.append('month', month);
+    formData.append('year', year);
+
+    // TODO get the csvFileBuffer from the csv in the user's session
+    const x = [1, 2, 3];
+    const buffer = Buffer.from(x);
```

Comment:
Still TODO as part of this ticket?

Reply:
Doing separately as a separate sub task francesca is doing